cmd/influxd/config.go

```diff
@@ -49,13 +51,6 @@ type Config struct {
 		Assets string `toml:"assets"`
 	} `toml:"admin"`
 
-	HTTPAPI struct {
-		Port        int      `toml:"port"`
-		SSLPort     int      `toml:"ssl-port"`
```

Comment:
I'm fine to put it in later. I'm mostly trying to clean up the config piece by piece since there's a lot of unused fields right now.